src/material/list/list.scss

```diff
@@ -162,13 +162,28 @@ a.mdc-list-item--activated {
   }
 }
 
-.mat-mdc-list-item.mdc-list-item--with-three-lines {
-  // List item lines or titles never wrap. MDC always enables wrapping for secondary text
-  // if the list item has acquired three lines. We unset these styles for line elements.
-  // https://github.com/material-components/material-components-web/blob/348665978ce73694ad4518626dd70cdf5b984113/packages/mdc-list/_evolution-mixins.scss#L205-L206.
-  // TODO: Consider removing once MDC supports the explicit tertiary line list variant.
+// Increased list item height if it has multiple lines so bottom line doesn't get
+// cut off. Also added padding-bottom so there's space btw the text and the divider.
+.mat-mdc-list-item.mdc-list-item--with-leading-avatar.mdc-list-item--with-two-lines,
+.mat-mdc-list-item.mdc-list-item--with-leading-checkbox.mdc-list-item--with-two-lines,
+.mat-mdc-list-item.mdc-list-item--with-leading-icon.mdc-list-item--with-two-lines,
+.mat-mdc-list-item.mdc-list-item--with-leading-avatar.mdc-list-item--with-three-lines,
+.mat-mdc-list-item.mdc-list-item--with-leading-checkbox.mdc-list-item--with-three-lines,
+.mat-mdc-list-item.mdc-list-item--with-leading-icon.mdc-list-item--with-three-lines {
+  height: auto;
+  padding-bottom: 16px;
```

Comment:
I think `16px` looks good, but `8px` will match the previous style and probably produce far less screenshot test failures